### PR TITLE
fixes #230

### LINKED ISSUE #230: NextJS - Type error when calling init functions
Hi.
I'm following the NextJS docs (https://supertokens.io/docs/emailpassword/nextjs/init), and on step 6 - "Call the init functions" we need to add the following function to the _app.ts file:

```ts
async function initNode() {
  const supertokensNode = await import('supertokens-node')
  const { backendConfig } = await import('../config/backendConfig')
  supertokensNode.init(backendConfig())
}
```

But I'm getting the following error on `supertokensNode.init(backendConfig())`:

```
Argument of type '{ framework: string; supertokens: { connectionURI: string; apiKey: string; }; appInfo: { appName: string; apiDomain: string; websiteDomain: string; apiBasePath: string; }; recipeList: RecipeListFunction[]; isInServerlessEnv: boolean; }' is not assignable to parameter of type 'TypeInput'.
  Types of property 'framework' are incompatible.
    Type 'string' is not assignable to type 'TypeFramework | undefined'.ts(2345)
```

Here's the backendConfig file that was auto-generated by the docs:

```ts
import EmailPasswordNode from "supertokens-node/recipe/emailpassword";
import SessionNode from "supertokens-node/recipe/session";
import { appInfo } from "./appInfo";

export const backendConfig = () => {
  return {
    framework: "express",
    supertokens: {
      // These are the connection details of the app you created on supertokens.io
      connectionURI:
        "<removed>",
      apiKey: "<removed>",
    },
    appInfo,
    recipeList: [EmailPasswordNode.init(), SessionNode.init()],
    isInServerlessEnv: true,
  };
};
```

I'm using supertokens-node@8.2.1

Thanks.

### PR DESCRIPTION
**Pull request by bhumilsarvaiya**
## Summary of change

Exposes types files

## Related issues

-   #230

## Checklist for important updates

-   [x] Changelog has been updated
-   [x] `coreDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `lib/ts/version.ts`
-   [x] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [x] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [x] Had run `npm run build-pretty`
-   [x] Had installed and ran the pre-commit hook
-   [x] Issue this PR against the latest non released version branch.
-   [x] If have added a new web framework, update the `add-ts-no-check.js` file to include that
-   [x] If added a new recipe / api interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).

## TODO:
- [ ] Edit docs for nextjs to use TypeInput in backendConfig function

### CODE REVIEW

**Comment by rishabhpoddar** [comment]
This should be issued against branch 8.3

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 .
 
+## [8.2.2] - 2021-12-20
```
Comment (rishabhpoddar) [inline CHANGELOG.md:9]: Against branch 8.3

File: types/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export * from "../lib/build/types";
```
Comment (rishabhpoddar) [inline types/index.d.ts:1]: Also, edit playground app in the test folder (it's called with-typescript) to make use of these types for all recipes and check that it's working as expected.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: types/index.d.ts (hunk 2 above)
Comment (bhumilsarvaiya) [reply types/index.d.ts:1]: done

File: CHANGELOG.md (hunk 1 above)
Comment (bhumilsarvaiya) [reply CHANGELOG.md]: done

**Comment by rishabhpoddar** [comment]
As discussed, types should be the end of the path, and not in the middle.

**Pull request merged**